android/build/filament-tasks.gradle

```diff
@@ -64,8 +64,8 @@ class MaterialCompiler extends DefaultTask {
     File matcPath
 
     @SuppressWarnings("GroovyUnusedDeclaration")
-    @InputDirectory
-    File inputDir
+    @InputFile
+    File inputFile
```

Comment:
The field is renamed, doesn't this break the script?

Reply:
The field name is not directly referenced in the script, I think this works because of the annotation.